src/components/create-courses/component/create-new-author.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {useState} from "react";
+import { v4 as uuidv4 } from 'uuid';
+import { Input, Label, Button, Row, InputGroup } from "reactstrap";
+
+export default function AddNewAuthor({ addAuthorToList }) {
+	//"Add New Author" functionality
+	const [newAuthorName, setNewAuthorName] = useState('');
+
+	function createNewAuthor(){
+		if (!/^[A-Z][A-Za-z]+\s+[A-Z][A-Za-z]/.test(newAuthorName)) {
+			return alert('Please enter your FullName');
+		}
+		const newAuthor = { id: uuidv4(), name: newAuthorName};
+		setNewAuthorName('')
+		return newAuthor
+	}
+
+	const newName = (event) => {
```

Comment:
Rename to onNameChange or onChangeHandler or something similar

Reply:
Done